JudoPayDotNetIntegrationTests/ConsumersTests.cs

```diff
@@ -12,23 +14,26 @@ namespace JudoPayDotNetIntegrationTests
     public class ConsumersTests
     {
         private JudoPayApi _judo;
+        private readonly Configuration _configuration = new Configuration();
+
+        [SetUp]
+        public void SetUp()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;
+        }
 
         [OneTimeSetUp]
         public void Init()
         {
-            _judo = JudoPaymentsFactory.Create(Configuration.Token,
-                Configuration.Secret,
-                Configuration.Baseaddress);
+            _judo = JudoPaymentsFactory.Create(_configuration.JudoEnvironment, _configuration.Token, _configuration.Secret);
         }
 
-       
-
         [Test]
         public void GetTransaction()
         {
             var paymentWithCard = new CardPaymentModel
             {
-                JudoId = Configuration.Judoid,
+                JudoId = _configuration.Judoid,
                 YourConsumerReference = "432438862",
                 Amount = 25,
                 CardNumber = "4976000000003436",
```

Comment:
This card number is being used in a lot of places. Can we abstract it into a test card container or similar?